src/app.js

```diff
@@ -1 +1,16 @@
+//---------- Firebase configuration starts from here
+import * as firebase from 'firebase'
+const firebaseConfig = {
+  apiKey: "AIzaSyAK10RzRApaDyJZfrs4NN-YSxaQvRZyOxg",
```

Comment:
We can add this api key in .env file.
We can download a npm package called dot-env.
Since the api key's are confidential, its always a good choice not to push it on github.